src/main.js

```diff
@@ -27,6 +27,7 @@ Vue.use(BootstrapVue)
 Vue.use(VueRouter)
 
 Vue.directive('access', {
+  isLiteral: true,
```

Comment:
Is there a way to not to include the element at all?

Reply:
should be the same as v-if directive